serverless/main.mk

```diff
@@ -31,8 +28,9 @@ ifeq (, $(SLS))
 	$(error "aws cli toolchain is not installed or incorrectly configured.")
 endif
 
-CMD_SLS_SERVICE_INSTALL = $(NPM) install --save-dev
-CMD_SLS_SERVICE_DEPLOY = $(SLS) deploy --env $(ENV) --region $(AWS_REGION) --profile $(AWS_PROFILE) --service $(SVC) -c $(SLS_FILE)
-CMD_SLS_SERVICE_BUILD = cd $(PROJECT_PATH) && make
-CMD_SLS_SERVICE_DESTROY = $(SLS) remove --env $(ENV) --region $(AWS_REGION) --profile $(AWS_PROFILE) --service $(SVC) -c $(SLS_FILE)
-CMD_SLS_SERVICE_SECRETS_PUSH = $(CMD_SLS_SERVICE_SECRETS_PUSH)
```

Comment:
I think this line is a bug. It should've been `CMD_SLS_SERVICE_SECRETS_PUSH  = $(CMD_SERVICE_SECRETS_PUSH)`, this way it would've worked when you run make secrets.

So let's keep it so we can use secrets push as well.

Reply:
fixed